shanoir-ng-users/src/main/java/org/shanoir/ng/user/service/UserService.java

```diff
@@ -111,6 +111,14 @@
 	@PreAuthorize("hasRole('ADMIN')")
 	List<User> getUsersToReceiveSecondExpirationNotification();
 
+	/**
+	 * Find users that are now expired.
+	 * 
+	 * @return a list of users.
+	 */
+	@PreAuthorize("hasRole('ADMIN')")
+	List<User> getExpiredUsers();
```

Comment:
Hi Jean-Come, thank you for your PR! What happens if a user is already expired, so its expirationDate is yesterday, but the code runs today? Will it be found too? Maybe we have a downtime for one or two days, would the code disable as well users, that expired yesterday?

Reply:
For the expiration date, we have no real way of tracking the "disabling" of a user. Maybe we take can a week of margin (I hope we'll never an interruption of more that 7 days !).
The "problem" (which is not really really a problem because we don't have millions of users) is that we'll update the same user in keycloak everyday for 7 days. Let's discuss this at the DSU.
Deletion is well pushed into keycloak.